addons/analytic/tests/test_analytic_account.py

```diff
@@ -71,6 +71,17 @@ def setUpClass(cls):
             'analytic_distribution': {cls.analytic_account_2.id: 100}
         })
 
+        def clean_access_rights(cls):
+            """ Removes all access right linked to timesheet and project as these
+            add record rules blocking analytic flows; account overrides it"""
+            if 'account.account' not in cls.env:
+                core_group_ids = cls.env.ref("project.group_project_manager", raise_if_not_found=False) | cls.env.ref("hr_timesheet.group_hr_timesheet_user", raise_if_not_found=False)
```

Comment:
The `|` doesn't work with `raise_if_not_found=False` because `None | None` doesn't exist (same by combining with one recordset)